### PR TITLE
Update release manifests for VPC CNI v1.11.1

### PR DESCRIPTION
**Pull request by orsenthil**
**What type of PR is this?**

- Updated release manifest in `Charts` and `config` directory.
- Replaced `v1.11.0` with `v1.11.1` in those directories.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: charts/aws-vpc-cni/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: aws-vpc-cni
 version: 1.1.16
```
Comment (jayanthvn) [inline charts/aws-vpc-cni/Chart.yaml:3]: Need to increment the chart version.

File: charts/cni-metrics-helper/Chart.yaml [hunk 2]
```diff
@@ -20,4 +20,4 @@ version: 0.1.8
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
-appVersion: v1.11.0
+appVersion: v1.11.1
```
Comment (jayanthvn) [inline charts/cni-metrics-helper/Chart.yaml:23]: Even here need to increment the chart version.

File: charts/aws-vpc-cni/Chart.yaml (hunk 1 above)
Comment (orsenthil) [reply charts/aws-vpc-cni/Chart.yaml:3]: Updated here - https://github.com/aws/amazon-vpc-cni-k8s/pull/1995 

branch protection prevented my update in this PR.

File: charts/cni-metrics-helper/Chart.yaml (hunk 2 above)
Comment (orsenthil) [reply charts/cni-metrics-helper/Chart.yaml:23]: Updated here - https://github.com/aws/amazon-vpc-cni-k8s/pull/1995 

branch protection prevented my update in this PR.

**Pull request closed without merging**